cc.config.json

```diff
@@ -25,17 +23,8 @@
         "3d": {
             "modules": ["3d"],
             "dependentAssets": [
-                "1baf0fc9-befa-459c-8bdd-af1a450a0319",
                 "511d2633-09a7-4bdd-ac42-f778032124b3",
-                "5d45aa00-e064-4938-b314-4265f0c2258c",
-                "28d6d6b8-3f66-4a73-9795-17a0852ba2d4",
```

Comment:
"28d6d6b8-3f66-4a73-9795-17a0852ba2d4" is "bloom.effect", move to custom pipeline.